cmd/influx_inspect/verify/tombstone/verify.go

```diff
@@ -0,0 +1,145 @@
+// Package tsm verifies integrity of TSM files.
+package tombstone
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/influxdata/influxdb/tsdb/engine/tsm1"
+)
+
+// Command represents the program execution for "influx_inspect verify-tombstone".
+type Command struct {
+	Stderr io.Writer
+	Stdout io.Writer
+}
+
+// NewCommand returns a new instance of Command.
+func NewCommand() *Command {
+	return &Command{
+		Stderr: os.Stderr,
+		Stdout: os.Stdout,
+	}
+}
+
+// Run executes the command.
+func (cmd *Command) Run(args ...string) error {
+	runner := verifier{w: cmd.Stdout}
+	fs := flag.NewFlagSet("verify-tombstone", flag.ExitOnError)
+	fs.StringVar(&runner.path, "path", os.Getenv("HOME")+"/.influxdb", "path to find tombstone files")
+	v := fs.Bool("v", false, "verbose: emit periodic progress")
+	vv := fs.Bool("vv", false, "very verbose: emit every tombstone entry key and time range")
+	vvv := fs.Bool("vvv", false, "very very verbose: emit every tombstone entry key and RFC3339Nano time range")
+
+	fs.SetOutput(cmd.Stdout)
+
+	if err := fs.Parse(args); err != nil {
+		return err
+	}
+
+	if *v {
```

Comment:
Could the complexity be simplified by checking `v` first, then `vv`, and finally `vvv`; using the most verbose option?

Would allow removal of complicated if statements

Reply:
That's what I said, but in a terrible fashion :)